src/main/java/org/jabref/gui/fieldeditors/LinkedFileViewModel.java

```diff
@@ -421,23 +421,22 @@ public void writeMetadataToPdf() {
         BackgroundTask<Void> writeTask = BackgroundTask.wrap(() -> {
             Optional<Path> file = linkedFile.findIn(databaseContext, preferences.getFilePreferences());
             if (file.isEmpty()) {
-                // TODO: Print error message
-                // Localization.lang("PDF does not exist");
+                dialogService.notify(Localization.lang("File not found"));
             } else {
                 try {
                     XmpUtilWriter.writeXmp(file.get(), entry, databaseContext.getDatabase(), preferences.getXmpPreferences());
 
                     EmbeddedBibFilePdfExporter embeddedBibExporter = new EmbeddedBibFilePdfExporter(preferences.getGeneralPreferences().getDefaultBibDatabaseMode(), Globals.entryTypesManager, preferences.getFieldWriterPreferences());
                     embeddedBibExporter.exportToFileByPath(databaseContext, databaseContext.getDatabase(), preferences.getFilePreferences(), file.get());
+                    
+                    dialogService.notify(Localization.lang("Finished writing metadata!"));
                 } catch (IOException | TransformerException ex) {
-                    // TODO: Print error message
-                    // Localization.lang("Error while writing") + " '" + file.toString() + "': " + ex;
+                    dialogService.notify(Localization.lang("Error while writing") + " '" + file.toString() + "': " + ex);
```

Comment:
Instead of printing the exception in the notify, I would add instead: dialogService.notify(Localization.lang("Error while writing metadata. See the error log for details.")